scene/resources/mesh_library.cpp

```diff
@@ -85,10 +85,10 @@ bool MeshLibrary::_get(const StringName &p_name, Variant &r_ret) const {
 		r_ret = get_item_mesh_transform(idx);
 	} else if (what == "shapes") {
 		r_ret = _get_item_shapes(idx);
-	} else if (what == "navmesh") {
```

Comment:
Should we leave the entries in `_get` and `_set` for the older names to allow for loading mesh libraries created using the old names?